packages/block-editor/src/components/index.js

```diff
@@ -155,6 +155,7 @@ export {
 export { default as __experimentalBlockPatternsList } from './block-patterns-list';
 export { default as __experimentalPublishDateTimePicker } from './publish-date-time-picker';
 export { default as __experimentalInspectorPopoverHeader } from './inspector-popover-header';
+export { default as __experimentalUseOnBlockDrop } from './use-on-block-drop';
```

Comment:
Should we mark this unstable? I don't think this is an API that we want to expose in this shape.

Reply:
Sure. I don't mind changing it to `__internal` or any other naming. I only use `__experimental` here because that's a more established pattern.